src/main/groovy/com/vanniktech/maven/publish/MavenPublishPlugin.groovy

```diff
@@ -78,7 +78,6 @@ class MavenPublishPlugin extends BaseMavenPublishPlugin {
     }.dependsOn("javadoc")
 
     configurer.addComponent(project.components.java)
-    configurer.addTaskOutput(project.jar)
```

Comment:
Is it really safe to not do this? Aren't we missing some of the task outputs for older versions?

Reply:
AFAIK the `jar` task has always been added to `archives` so this should be safe.